src/locales/fr.json

```
@@ -123,22 +251,81 @@
     "room_label": "Salle :",
     "start_delay_seconds": "Temps démarrage :",
     "stop_delay_seconds": "Temps arrêt :",
-    "room_id_placeholder": "Ex: room-1"
+    "room_id_placeholder": "Ex: room-1",
```

Biome (1.9.4)

[error] The key room_label was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)